warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/TableSplitsCache.java

```diff
@@ -211,21 +268,36 @@ private boolean exceedsMaxSplitsDeviation(Map<String,Integer> tmpSplitsPerTable)
     @Override
     protected void readCache(BufferedReader in, String delimiter) throws IOException {
```

Comment:
Same comment about the delimiter arg as for the other pull request

Reply:
I planned to address this once the other MR is merged